Add 5.14 release notes (#1795)
sensu/sensu-docs: content/sensu-go/5.14/release-notes.md

---

## 5.14.0 release notes

> **October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes bug fixes that affect round robin schedule recovery after quorum loss and enhanced TSL authentication support.

Reviewer: "affect" or "enable"? I'm wondering if we can include stronger language around there being a fix.

Reviewer: also, should it be TLS authentication?

Author: Yes to both, thank you!

> **October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes bug fixes that affect round robin schedule recovery after quorum loss and enhanced TSL authentication support.

Reviewer 2: s/TSL/TLS

Author: Thank you!

> **October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes bug fixes that affect round robin schedule recovery after quorum loss and enhanced TSL authentication support.

Author: ```suggestion
**October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes bug fixes that enable round robin schedule recovery after quorum loss and enhanced TLS authentication support.
```

See the [upgrade guide][1] to upgrade Sensu to version 5.14.0.

**NEW FEATURES:**

- The [web UI][80] now includes an error dialog option that allows users to wipe the application's persisted state (rather than having to manually wipe their local/session storage). This can help in the rare cases that something in the persisted state is leading to an uncaught exception.
- The [web UI][80] now respects the system preference for operating systems with support for selecting a preferred light or dark theme.
- `sensuctl dump` can now list the types of supported resources with `sensuctl dump --types`.
- The [entity resource][82] now includes the `sensu_agent_version` field, which reflects the Sensu Semantic Versioning (SemVer) version of the agent entity.

> - There are two new configuration options for backends using the embedded etcd: `--etcd-heartbeat-interval` and `etcd-election-timeout`.

Author: ```suggestion
- There are two new configuration options for `sensu-backend` using embedded etcd: `--etcd-heartbeat-interval` and `etcd-election-timeout`.
```

> - There are two new configuration options for `sensu-backend` using embedded etcd: `--etcd-heartbeat-interval` and `etcd-election-timeout`.

Author: ```suggestion
- There are two new configuration options for `sensu-backend` using embedded etcd: `etcd-heartbeat-interval` and `etcd-election-timeout`.
```

**IMPROVEMENTS:**

- ([Licensed tier][79]) Added support for mutual TLS authentication between agents and backends.
- ([Licensed tier][79]) Added support for CRL URLs for mTLS authentication.
- To facilitate discussion about feature requests, in the Web UI, feedback is directed to Discourse rather than the GitHub repository's issues page.
- In the [web UI][80], when a user lands on a page inside a namespace that no longer exists or they do not have access to, the drawer opens to that namespace switcher to help clarify next steps.

> - Support agent [TLS authentication][81] is usable with a licensed sensu-backend.

Reviewer: Suggest grouping the TLS improvements together (so making this third on the list)

> - Updated Sensu Go version from 1.12.3 to 1.13.1.

Reviewer 3: It's the Go (golang) version that was updated, not Sensu Go!

Author: 🤦‍♀ thank you!

> - Updated Sensu Go version from 1.12.3 to 1.13.1.

Author: ```suggestion
- Updated Go version from 1.12.3 to 1.13.1.
```

**FIXES:**

- ([Licensed tier][79]) `sensuctl` can now create federation resources.

> - ([Licensed tier][79]) `sensuctl` on Windows can now create postgres resources.

Reviewer: suggest capitalizing Postgres here and below

- ([Licensed tier][79]) Fixed a bug that resulted in event metrics being ignored when using the Postgres store.

> - Splayed proxy checks are executed every interval (instead of every interval + interval * splay_coverage).

Author: ```suggestion
- Fixed a bug that caused checks to stop executing after a network error.
- Splayed proxy checks are executed every interval (instead of every interval + interval * splay_coverage).
```

---

## 5.14.0 release notes

> **October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes bug fixes that enable round robin schedule recovery after quorum loss and enhanced TLS authentication support.

Author: ```suggestion
**October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes enhanced TLS authentication support and bug fixes that restore check execution after a network error and enable round robin schedule recovery after quorum loss.
```

> **October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes enhanced TLS authentication support and bug fixes that restore check execution after a network error and enable round robin schedule recovery after quorum loss.

Author: ```suggestion
**October 8, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes enhanced TLS authentication support and bug fixes that restore check execution after a network error and enable round robin schedule recovery after quorum loss.
```

---

## 5.14.0 release notes

**October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download.
There are some exciting feature additions in this release, including .... Additionally, this release includes bug fixes for ... and enhanced ...
See the [upgrade guide][1] to upgrade Sensu to version 5.14.0.

**NEW FEATURES:**

- The [web UI][80] now includes an error dialog option that allows users to wipe the application's persisted state (rather than having to manually wipe their local/session storage). This can help in the rare cases that something in the persisted state is leading to an uncaught exception.
- The [web UI][80] now respects the system preference for operating systems with support for selecting a preferred light or dark theme.
- `sensuctl dump` can now list the types of supported resources with `sensuctl dump --types`.

> - The Entity resource now includes the `sensu_agent_version` field, which reflects the Sensu semver version of the agent entity.

Author: ```suggestion
- The [entity resource][82] now includes the `sensu_agent_version` field, which reflects the Sensu semver version of the agent entity.
```

- .
- .

**IMPROVEMENTS:**

- ([Licensed tier][79]) Added support for mutual TLS authentication between agents and backends.
- ([Licensed tier][79]) Added support for CRL URLs for mTLS authentication.
- To facilitate discussion about feature requests, in the Web UI, feedback is directed to Discourse rather than the GitHub repository's issues page.
- In the [web UI][80], when a user lands on a page inside a namespace that no longer exists or they do not have access to, the drawer opens to that namespace switcher to help clarify next steps.

> - Support agent TLS authentication is usable with a licensed sensu-backend.

Author: ```suggestion
- Support agent [TLS authentication][81] is usable with a licensed sensu-backend.
```

[76]: /sensu-go/5.11/sensuctl/reference#view-sensuctl-config
[77]: /sensu-go/5.11/reference/backend#advanced-configuration-options
[78]: /sensu-go/5.12/reference/agent/#allow-list
[79]: /sensu-go/5.14/getting-started/enterprise

> [80]: /sensu-go/5.14/dashboard/overview

Author: ```suggestion
[80]: /sensu-go/5.14/dashboard/overview
[81]: /sensu-go/5.14/guides/securing-sensu#sensu-agent-tls-authentication
```

[76]: /sensu-go/5.11/sensuctl/reference#view-sensuctl-config
[77]: /sensu-go/5.11/reference/backend#advanced-configuration-options
[78]: /sensu-go/5.12/reference/agent/#allow-list
[79]: /sensu-go/5.14/getting-started/enterprise
[80]: /sensu-go/5.14/dashboard/overview

> [81]: /sensu-go/5.14/guides/securing-sensu#sensu-agent-tls-authentication

Author: ```suggestion
[81]: /sensu-go/5.14/guides/securing-sensu#sensu-agent-tls-authentication
[82]: /sensu-go/5.13/reference/entities/
```
